src/components/TemplateDetails/styles.scss

```diff
@@ -0,0 +1,90 @@
+.whatsapp_template_summary {
+  &__title {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
```

Comment:
Use unnnic token, not just in this place but anywhere where a value exists in the unnnic specification.